internal/pkg/composable/providers/kubernetes/hints_test.go

```diff
@@ -367,6 +367,149 @@ func TestGenerateHintsMappingWithLogStream(t *testing.T) {
 	assert.Equal(t, expected, hintsMapping)
 }
 
+func TestGenerateHintsMappingWithDefaultsandTypo(t *testing.T) {
+	logger := getLogger()
+	pod := &kubernetes.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "testpod",
+			UID:       types.UID(uid),
+			Namespace: "testns",
+			Labels: map[string]string{
+				"foo":        "bar",
+				"with-dash":  "dash-value",
```

Comment:
is the different format of labels relevant for this test?

Reply:
No, copy paste from other test

I removed them in latest commit